meinberlin/assets/scss/_variables.scss

```diff
@@ -9,6 +9,7 @@ $brand-danger: #dc3918;
 
 $body-bg: #fff;
 $bg-secondary: #f6f6f6;
+$bg-tertiary: #e6e6e6;
```

Comment:
We should not add new color variables all the time. For me it is not clear when to use which.